jvb/src/test/kotlin/org/jitsi/videobridge/rest/root/colibri/mucclient/MucClientTest.kt

```diff
@@ -91,25 +92,37 @@ class MucClientTest : JerseyTest() {
     fun testRemoveMuc() {
         val jsonConfigSlot = slot<JSONObject>()
 
-        every { xmppConnection.addMucClient(capture(jsonConfigSlot)) } returns true
+        every { xmppConnection.removeMucClient(capture(jsonConfigSlot)) } returns true
 
         val json = JSONObject().apply {
             put("id", "id")
         }
 
-        val resp = target("$baseUrl/add").request().post(Entity.json(json.toJSONString()))
+        val resp = target("$baseUrl/remove").request().post(Entity.json(json.toJSONString()))
         resp.status shouldBe HttpStatus.OK_200
         jsonConfigSlot.captured shouldBe json
     }
 
     @Test
     fun testRemoveMucFailure() {
-        every { xmppConnection.addMucClient(any()) } returns false
+        every { xmppConnection.removeMucClient(any()) } returns false
 
         val json = JSONObject().apply {
             put("id", "id")
         }
-        val resp = target("$baseUrl/add").request().post(Entity.json(json.toJSONString()))
+        val resp = target("$baseUrl/remove").request().post(Entity.json(json.toJSONString()))
         resp.status shouldBe HttpStatus.BAD_REQUEST_400
     }
+
+    @Test
+    fun testGetIds() {
+        val fakeIds = JSONArray().apply {
+            add("id1")
+        }
+        every { xmppConnection.getMucClientIds() } returns fakeIds.toJSONString()
+
+        val resp = target("$baseUrl/list").request().get()
+        resp.readEntity(String::class.java) shouldBe "[\"id1\"]"
+    }
+
```

Comment:
The extra whitespace triggers a ktlint error, you can fix it automatically with `ktlint -F`

Reply:
Thanks. will fix.